### PR TITLE
Update Terraform Documentation. Incl Experiments

### PR DESCRIPTION
**Pull request by daniel-statsig**
# DO NOT Publish until 2.0 is released (And update Links)

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://6402cfa12ae9336d653b83da--cozy-fox-0defec.netlify.app

File: docs/integrations/terraform/introduction.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Statsig Terraform Provider
+---
+
+The Statsig Terraform Provider allows you to configure your gates and experiments with Terraform. This is achieved by leveraging the Console API. If there is something you need to perform that isn't supported by the Terraform Provider, checkout the [Console API](https://docs.statsig.com/console-api/introduction).
```
Comment (tore-statsig) [inline docs/integrations/terraform/introduction.md:5]: This is achieved by leveraging the Console API => The provider synchronizes with Statsig via the console API
Or something.  "This is achieved by leveraging" is just awkward

File: docs/integrations/terraform/introduction.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+It is hosted on the Terraform registry at https://registry.terraform.io/providers/statsig-io/statsig
+
+## Supported Features
+
+We currently only support a few Statsig configuration via Terraform, but are working on adding more.
```
Comment (tore-statsig) [inline docs/integrations/terraform/introduction.md:11]: We currently only support the following Statsig configurations:

We will be adding Segments and Dynamic Configs shortly

Or something like that

File: docs/integrations/terraform/terraform_experiment.mdx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## Changing Experiment Status
+
+You can update the `status` field to four possible values, **setup**, **active**, **decision_made** and **abandoned**.
+
+If you would like to see code examples of how the **Setup -> Run -> Ship** flow works for an experiment, check out our [Terraform Acceptance Tests](#) for experiments.
+
+#### Status: setup
+
+When an experiment has this status, you are stating that your experiment is not ready, and no values will be served via a Statsig SDK or the HttpAPI.
```
Comment (tore-statsig) [inline docs/integrations/terraform/terraform_experiment.mdx:53]: well...unless someone is overridden

File: docs/integrations/terraform/terraform_experiment.mdx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+#### Status: abandoned
+
+Experiments with this status will not serve any values and will not collect any analytics information.
+
+:::info
+
+- You may only create an experiment with the status "setup" or "active".
+
+- You can only transition to "decision_made" from "setup".
```
Comment (tore-statsig) [inline docs/integrations/terraform/terraform_experiment.mdx:72]: uh...you mean from "active"?

File: docs/integrations/terraform/terraform_experiment.mdx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+:::
+
+A full experiment example is included in the open source Github repo https://github.com/statsig-io/terraform-provider-statsig/todo
```
Comment (tore-statsig) [inline docs/integrations/terraform/terraform_experiment.mdx:76]: neato

**Review by tore-statsig — COMMENTED** [review]
some updates

**Pull request merged**